configure.ac

```diff
@@ -1247,27 +1251,13 @@ else
 fi
 AM_CONDITIONAL(BGPD, test "x$BGPD" = "xbgpd")
 
-if test "${enable_ripd}" = "no";then
-  RIPD=""
-else
-  RIPD="ripd"
-fi
-AM_CONDITIONAL(RIPD, test "x$RIPD" = "xripd")
+AM_CONDITIONAL(RIPD, test "${enable_ripd}" != "no")
```

Comment:
- AM_CONDITIONAL is a boolean
- `${enable_ripd}` has 3 useful values:
  - `""` - neither `--enable-ripd` nor `--disable-ripd` was specified
  - `"yes"` - results from `--enable-ripd`, `--enable-ripd=yes` (and maybe `--disable-ripd=no`)
  - `"no"` - results from `--enable-ripd=no`, `--disable-ripd`

In theory you can specify `--enable-ripd=foobar`, but the value isn't useful here.

Ultimately, just going with `"${enable_ripd}" != "no"` does the right thing in that it enables ripd by default and you can disable it.